app/jobs/update_github_repos_job.rb

```diff
@@ -0,0 +1,113 @@
+class UpdateGithubReposJob < CourseJob
+  @job_name = "Update GitHub Repository Info"
+  @job_short_name = "update_github_info"
+  @job_description = "Uses smart querying to quickly update GitHub repositories and their collaborators."
+
+  def attempt_job(course_id)
+    course = Course.find(course_id)
+    course_student_users = course.roster_students.map { |rs| rs.user }.select { |u| u != nil }
```

Comment:
Does `.compact` work as well as `.select { |u| u != nil}`?

Reply:
You're right, I didn't know about compact. I'll change that now

Reply:
Done in 020e420